custom/icds_reports/utils.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, domain, filters, report_id):
         self.report_config = report_config
 
     def data(self):
-        return self.report_config.get_data()
+        return self.report_config.data_source._get_data().values()
```

Comment:
How does this improve performance? It looks like it's going to be loading the entire UCR data source in memory instead of just what it needs for this report